src/app/views/company/team-list/modals/member-add-modal.controller.coffee

```diff
@@ -0,0 +1,34 @@
+angular.module 'mnoEnterpriseAngular'
+  .controller('MemberAddModalCtrl',
+  (MnoeTeams, $uibModalInstance, Utilities, team, availableUsers) ->
+    'ngInject'
+
+    vm = this
+    vm.team = team
+    vm.availableUsers = availableUsers
+    vm.selectedUsers = []
+    vm.isLoading = false
+
+    vm.toggleUser = (user) ->
+      if _.includes(vm.selectedUsers, user) then _.remove(vm.selectedUsers, user) else vm.selectedUsers.push(user)
```

Comment:
Leverage coffeescript rather than lodash if possible:
```coffeescript
if user in vm.selectedUsers then ...
```
This will be more efficient as coffescript will be compiled to javascript and you don't have the overhead of lodash.
Have a look at this for more info: https://arcturo.github.io/library/coffeescript/04_idioms.html

This also has the benefit of being easier to read ;)